### PR TITLE
Implement coordinate transformations

### PR DESCRIPTION
**Pull request by zichunhao**

### CODE REVIEW

File: jetnet/utils/utils.py [hunk 1]
```diff
@@ -148,3 +149,202 @@ def to_image(
...
+    idx_eta, idx_phi, idx_pt = 0, 1, 2
+    pt = jet[..., idx_pt]
+    eta = jet[..., idx_eta]
+    phi = jet[..., idx_phi]
+
+    num_particles = jet.shape[-2]
+    if isinstance(jet, np.ndarray):
+        pt /= np.repeat(np.expand_dims(jet_features[..., idx_pt], axis=-1), num_particles, axis=-1)
+        eta -= np.repeat(np.expand_dims(jet_features[..., idx_eta], axis=-1), num_particles, axis=-1)
+        phi -= np.repeat(np.expand_dims(jet_features[..., idx_phi], axis=-1), num_particles, axis=-1)
```
Comment (rkansal47) [inline jetnet/utils/utils.py:344]: this doesn't take edge cases into account where phi ends up being less than `pi`.

The correct formula is here https://github.com/CoffeaTeam/coffea/blob/master/coffea/nanoevents/methods/vector.py#L67-L68

File: jetnet/utils/utils.py [hunk 2]
```diff
@@ -148,3 +149,202 @@ def to_image(
...
+        return np.stack((eta, phi, pt), axis=-1)
+
+    pt /= jet_features[..., idx_pt].unsqueeze(dim=-1).repeat(1, num_particles)
```
Comment (rkansal47) [inline jetnet/utils/utils.py:347]: Even though they have the same effect, I'd turn this into an if elif block still with `elif isinstance(jet, torch.Tensor` just to be more clear.

File: jetnet/utils/utils.py [hunk 3]
```diff
@@ -148,3 +149,202 @@ def to_image(
             jet_image[phi, eta] += pt
 
     return jet_image
+
+
+def get_polar(
```
[the remark this answers is not in the document]
Comment (zichunhao) [reply jetnet/utils/utils.py:154]: Should we have a more general case of `cartesian_to_polar`, where mass is a function parameter?

[the remark this answers is not in the document]
Comment (rkansal47) [reply jetnet/utils/utils.py:154]: That would be nice, yeah

**Pull request closed without merging**